doc/compile.txt

```diff
@@ -18,3 +20,8 @@ Builds the app for specified platforms, or all platforms
     --target ........................... Deploy to a specific target
 
 To provide platform specific options, you must include them after `--`.
+
+Examples
+
+    cordova-cli compile android windows --debug --device
+    cordova-cli compile android --release
```

Comment:
It would be a good idea to include an example for one of the platform specific options